draft-ietf-rtcweb-jsep.xml

```diff
@@ -3929,13 +3920,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             that the JSEP implementation should be sending media
             ("sendonly" for local answers, "recvonly" for remote
             answers, or "sendrecv" for either type of answer),
-            choose the media format to send
-            as the most preferred media format from the remote
-            description that is also present in the answer, as
-            described in
-            <xref target="RFC3264" />, Sections 6.1 and 7, and start
-            transmitting RTP media once the underlying transport layers
-            have been established. If an SSRC has not already been
+            choose the media format to send; for the offerer, this will be
+            the most preferred media format from the answer that is also
+            locally supported, as described in <xref target="RFC3264" />,
+            Section 7; for the answerer, this will be
```

Comment:
This is a non realistic case but I hope it helps illustrate ...  Lets say Alice can only receive 711 and 729 (prefers 729) but can also send opus (but can't receive it). Alice offers 729, 711 in that order. Bob can do all all of the above and prefers in order opus, 729, 711. Bob relies in answer with, assuming a preferences override, SDP that says opus,  729 , 711. Alice sends opus as that was bobs top choice. Bob sends 729 as that was Alice top choice.  Does that outcome seem right? (Does the spec allow an app to cause that to happen? I think it does if caller prefs are explicitly set they do, but if not set, then Bob's answer will only have 729, 711, opus and Alice will send 729 which seems like a reasonable default.